### DISCUSSION TITLE
How to customise tooltip for within foliumap module?

### DISCUSSION THREAD

**Discussion by krvrd** [opengeos/leafmap#365] [Q&A] [answered]
I have a layer added with add_data() method and I would like to customise name aliases and count of info to render. How I can do that?

**Comment by giswqs** [comment, marked as answer]
The `data` parameter accepts a Pandas DataFrame. You can customize the DataFrame by changing the column names.

**Reply by krvrd** [reply]
Thank you for answer. Could I hide two service columns **category** and **color** which generated by method?

**Comment by giswqs** [comment]
I have added a `fields` parameter for customizing the columns in the tooltip. See https://github.com/giswqs/leafmap/pull/366. Run `leafmap.update_package()` and restart the kernel to take effect.
